packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -52,19 +52,14 @@ export class NgccProcessor {
 
   /** Process the entire node modules tree. */
   process() {
-    const timeLabel = 'NgccProcessor.process';
-    time(timeLabel);
-
-    const corePackage = this.tryResolvePackage('@angular/core', this._nodeModulesDirectory);
-
-    // If the package.json is read only we should skip calling NGCC.
-    // With Bazel when running under sandbox the filesystem is read-only.
-    if (corePackage && isReadOnlyFile(corePackage)) {
```

Comment:
I think it might be good to keep this as a check for other potential cases of read-only node modules directories.

Reply:
Good point, added back.